sql/dql/basic-queries/where-5.md

```diff
@@ -0,0 +1,163 @@
+| Squirtle  | 314   | 44 | 48     | 65      |
+| Wartortle | 405   | 59 | 63     | 80      |
+
+You can use the `BETWEEN` operator[2] to define a range of acceptable values, like so:
+
+```sql
+SELECT *
+FROM pokemon
+WHERE attack BETWEEN 60 AND 100;
+```
+
+The resulting set will be:
+
+| name       | total | hp | attack | defense |
+|------------|-------|----|--------|---------|
+| Ivysaur    | 405   | 60 | 62     | 63      |
+| Venusaur   | 525   | 80 | 82     | 83      |
+| Charmeleon | 405   | 58 | 64     | 58      |
+| Charizard  | 534   | 78 | 84     | 78      |
+| Wartortle  | 405   | 59 | 63     | 80      |
+| Blastoise  | 530   | 79 | 83     | 100     |
+
+---
+## Practice
+
+Select all the columns from the `pokemon` table, keeping only the rows where the name ends with the string `'saur'`.
+
+```sql
+SELECT * 
+FROM pokemon
+??? ??? ??? ???;
+```
+
+* WHERE
+* name
+* LIKE
+* '%saur'
+* NAME IS
+* AND
+* 'saur'
+* 'saur%'
+
+---
+## Revision
+
+Select all columns from the `pokemon` table, that have an `attack` in the range `65-75`.
+
+```sql
+SELECT *
+FROM pokemon
+??? ??? ??? ??? ??? ???;
+```
+
+* WHERE
+* attack
+* BEWTEEN
+* 65
+* AND
+* 75
+
+
+---
+## Quiz 
+### Do you know how to match a pattern in SQL?
+Consider the `location` table from our pokemon database:
+
+| id  | region_id | name          |
+|-----|-----------|---------------|
+| 1   | 4         | canalave-city |
+| 2   | 1         | eterna-city   |
+| 3   | 3         | pastoria-city |
+| ... | ...       | ...           |
+
+Choose the query that will get all the location records that are based in regions 1, 2. Also, pay attention to include only locations that end in 'city':
+
+ ???
+
+* SELECT * FROM location WHERE region_id < 3 AND name LIKE '%city';
+* SELECT * FROM location WHERE region_id < 3 AND name='%city';
+* SELECT * FROM location WHERE BETWEEN 1 AND 2 AND name='%city';
+* SELECT * FROM location WHERE region_id < 3 AND name LIKE '_city';
+
+---
+## Footnotes
+[1:RDBMS Implementations]
+Note that some databases may have a different implementation of the specific wildcard characters
+[2: BETWEEN]
+This can be seen as a shortcut for chaining two comparison operators with the `AND` clause:
+```sql
+SELECT * FROM table_name
+WHERE col BETWEEN 1 AND 10;
+-- is the same as
+SELECT * FROM table_name
+WHERE col >= 1 AND
+  col <= 10;
+```
+[3:Table]
+| name       | total | hp | attack | defense |
+|------------|-------|----|--------|---------|
+| Bulbasaur  | 318   | 45 | 49     | 49      |
+| Ivysaur    | 405   | 60 | 62     | 63      |
+| Venusaur   | 525   | 80 | 82     | 83      |
+| Charmander | 309   | 39 | 52     | 43      |
+| Charmeleon | 405   | 58 | 64     | 58      |
+| Charizard  | 534   | 78 | 84     | 78      |
+| Squirtle   | 314   | 44 | 48     | 65      |
+| Wartortle  | 405   | 59 | 63     | 80      |
+| Blastoise  | 530   | 79 | 83     | 100     |
+| Caterpie   | 195   | 45 | 30     | 35      |
+| Metapod    | 205   | 50 | 20     | 55      |

```

Comment:
``````suggestion
[1:Table]
| name       | total | hp | attack | defense |
|------------|-------|----|--------|---------|
| Bulbasaur  | 318   | 45 | 49     | 49      |
| Ivysaur    | 405   | 60 | 62     | 63      |
| Venusaur   | 525   | 80 | 82     | 83      |
| Charmander | 309   | 39 | 52     | 43      |
| Charmeleon | 405   | 58 | 64     | 58      |
| Charizard  | 534   | 78 | 84     | 78      |
| Squirtle   | 314   | 44 | 48     | 65      |
| Wartortle  | 405   | 59 | 63     | 80      |
| Blastoise  | 530   | 79 | 83     | 100     |
| Caterpie   | 195   | 45 | 30     | 35      |
| Metapod    | 205   | 50 | 20     | 55      |

[2:RDBMS Implementations]
Note that some databases may have a different implementation of the specific wildcard characters

[3: BETWEEN]
This can be seen as a shortcut for chaining two comparison operators with the `AND` clause:
```sql
SELECT * FROM table_name
WHERE col BETWEEN 1 AND 10;
-- is the same as
SELECT * FROM table_name
WHERE col >= 1 AND
  col <= 10;
```
``````